variables.tf

```diff
@@ -5,7 +5,7 @@ variable "create_keys" {
 }
 
 variable "keys" {
-  description = "Schema list of KMS Keys, consisting of alias, (OPTIONAL) description, (OPTIONAL) deletion_window_in_days, (OPTIONAL) policy"
+  description = "Schema list of KMS Keys, consisting of alias, (OPTIONAL) description, (OPTIONAL) deletion_window_in_days, (OPTIONAL) policy, (OPTIONAL) enable_key_rotation"
   type        = list
```

Comment:
guessing this should be a bool?

Reply:
nvm lol!

Reply:
Yes, it should be a boolean. I can add that to the description.